TinCanTests/ActivityTest.cs

```diff
@@ -40,12 +40,16 @@ public void TestActivityIdCase()
         }
 
         [Test]
-        [ExpectedException("System.UriFormatException")]
         public void TestActivityIdInvalidUri()
         {
-            var activity = new Activity();
-            string invalid = "foo";
-            activity.id = invalid;
+            Assert.Throws<System.UriFormatException>(
```

Comment:
So this is preferable to `Assert.That(() => UseActivityIdInvalidUri(), Throws.InstanceOf<System.UriFormatException>());`?

It was suggested to me by @mlefoster that `Assert.That` was a more common NUnit 3 syntax.